source/chapters/library.rst

```diff
@@ -866,6 +867,86 @@ different location. Select :guilabel:`Use Default Library` to reset.
   Mac partition mounted in Linux, you can load it and use your iTunes tracks and
   playlists as well.
 
+.. _import-serato-metadata:
+
+Importing metadata from Serato
+------------------------------
+
+From Mixxx 2.3, tracks in your Serato library and crates can now be imported into Mixxx library table.
+Tracks can also be loaded into deck without needing to add the music directory in the preferences. 
+You can also import your Serato library into Mixxx that are in USB drives for a portable Serato library.
```

Comment:
```suggestion
Mixxx supports reading your Serato library and crates both from your local hard drive as well as portable USB drives. Smart crates are currently not supported.

Serato stores information such as hot cues, track color and beatgrid in the file tags.
Regardless of whether a track is loaded into deck from the libray or directly from the file browser, Mixxx will import that information automatically if present.

Saved loops are imported as well, but since Mixxx is lacking support for saved loops, only the first loop is usable in Mixxx.
All other saved loops can be used like regular hotcues for now.
   
Waveforms, Gain values and "Flips" are not imported from Serato.
   
.. note:: Due to differences in the metadata format, importing Serato information from Ogg Vorbis files is currently not supported.
```